vm_supervisor/metrics.py

```diff
@@ -34,6 +34,8 @@ class ExecutionRecord(Base):
     __tablename__ = "records"
 
     uuid = Column(String, primary_key=True)
+
+    execution_uuid = Column(String, nullable=False)
```

Comment:
You recreate the DB on startup, right? Otherwise adding a column will crash.

Reply:
Indeed, the database would require a migration.